core/controllers/feedback.py

```diff
@@ -59,11 +59,12 @@ class ThreadHandler(base.BaseHandler):
 
     def get(self, exploration_id, thread_id):  # pylint: disable=unused-argument
         self.values.update({
-            'messages': feedback_services.get_messages(
-                exploration_id, thread_id)})
+            'messages': [m.to_dict() for m in feedback_services.get_messages(
+                exploration_id, thread_id)]})
+        suggestion = feedback_services.get_suggestion(exploration_id, thread_id)
```

Comment:
Small refactoring suggestion: maybe declare suggestion at the top of get(), and then do a single self.values.update() with both 'messages' and 'suggestion'? Seems cleaner.